### PR TITLE
journalctl: add errors from this boot example

### PR DESCRIPTION
**Pull request by tom-on-the-internet**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/journalctl.md [hunk 1]
```diff
@@ -10,6 +10,10 @@
 
 `journalctl -b -1`
 
+- Show all errors from this boot (-p is priority and 3 is for errors):
```
Comment (navarroaxel) [inline pages/linux/journalctl.md:13]: Is this better?
```suggestion
- Show all messages with priority level 3 (errors) from this boot:
```

File: pages/linux/journalctl.md [hunk 2]
```diff
@@ -10,6 +10,10 @@
...
+
+`journalctl -p 3 -b`
```
Comment (navarroaxel) [inline pages/linux/journalctl.md:15]: We can use long argument format to be easier for understand, and the argument option should be in `{{}}`
```suggestion
`journalctl -b --priority={{3}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @tom-on-the-internet welcome and thanks for your submission. I left some comments.

**Comment by tom-on-the-internet** [comment]
> Hey @tom-on-the-internet welcome and thanks for your submission. I left some comments.

Thanks so much!
Your suggestions look great.

**Review by navarroaxel — APPROVED** [review]
LGTM, thanks @tom-on-the-internet

**Review by sbrl — APPROVED** [review]
Thanks, @tom-on-the-internet! Looks good to me.

**Pull request merged**